python/vowpalwabbit/pyvw.py

```diff
@@ -597,11 +597,16 @@ def __init__(self, costs=[], prediction=0):
     def from_example(self, ex):
         class wclass:
             def __init__(self,
-                         label,
+                         action=None,
                          cost=0.,
                          partial_prediction=0.,
-                         probability=0.):
-                self.label = label
+                         probability=0.,
+                         **kwargs):
+                if kwargs.get('label', False):
+                    action = kwargs['label']
```

Comment:
If action and label are both passed in, label will overwrite action. Is that the behavior we want?

Reply:
Actually, I don't think that it might happen as there are two possibilities if any user is familiar with is old format then he might use them as below,
First,
`>>> wc = wclass(paramter1, parameter2, paramter3, parameter4)`
or second,
```
>>> wc = wclass(label=paramter1, cost=paramter2, partial_prediction=paramter3, probability=paramter4)
```
If someone knowing the current format would use the below two formats
`>>> wc = wclass(paramter1, parameter2, paramter3, parameter4)`
or,
```
>>> wc = wclass(action=paramter1, cost=paramter2, partial_prediction=paramter3, probability=paramter4)
```
So the all 4 cases works fine with the current implementation.